### DISCUSSION TITLE
example related to "ElasticFoundation"

### DISCUSSION THREAD

**Discussion by ebrahimraeyat** [xcfem/xc#119] [Q&A] [answered]
Hello @lcpt . As [xc_examples](https://github.com/xcfem/xc_examples) did not Discussions, I asked it here:

Is there any example related to ElasticFoundation class? I have written my [OSAFE](https://github.com/ebrahimraeyat/OSAFE) workbench in FreeCAD and I can model the foundation with it. I wanted to work with creating xc foundation model from FreeCAD model (IFC) and then analyzing the foundation. I will share the work in github.
Thanks

**Comment by lcpt** [comment, marked as answer]
I've, enabled discussions in this repository :)

Yes, there is an example [here](https://github.com/xcfem/xc_examples/blob/master/reinforced_concrete/water_channel/xc_model.py).

For cases like this, you can search for the class name string in the repository files.

**Comment by ebrahimraeyat** [comment]
Yes, I searched in repository and also in Google, but did not find any result! Thanks @lcpt .
I wrote an example and I wanted to verify it.

**Comment by lcpt** [comment]
That's weird. Have you tried something like this (in your local XC folder)?:

```grep --include=\*.py -rnw './' -e 'ElasticFoundation'```

That must show you all the occurrences of that string in the Python code.

**Reply by ebrahimraeyat** [reply]
> grep --include=\*.py -rnw './' -e 'ElasticFoundation'

Thanks. It gives me two results, in python modules and build folder.
